.travis.yml

```diff
@@ -36,13 +38,18 @@ env:
     - secure: oGdTYh3rnPdG7qVGYq3TgVESypir1btO2kmafumtKR6o+FYmMtb09pe+HtiT6Qfv4pDwnYZ2RTYqsTY4h6POigU5j7qQPSsZ5WkhHt9ybFyt/vfo2B95GaAIFt+8LVjTX5TItyw0/PPUCCUluQ1n8k/NCtSfMb0kIT/CuLrg7N8=
 
 before_script:
+  # PostgreSQL-10 is still not fully supported by Travis.
```

Comment:
This post (https://github.com/travis-ci/travis-ci/issues/8537#issuecomment-399738282) says that the approach described in Travis docs does indeed work (https://docs.travis-ci.com/user/database-setup/#using-a-different-postgresql-version). All you have to do is use a port other than 5432.

Reply:
Yes, I saw this. It came down to these few lines in .travis.yml, or providing a travis-specific Django settings file pointing to the non-standard port, and I chose this workaround.

Presumably when https://github.com/travis-ci/travis-ci/issues/8537 is closed, the travis-provided postgresql will also operate on the default 5432 port.

Reply:
OK but this adds to technical debt of having separate/duplicated provisioning code on Travis (see #822).